### PR TITLE
Change host spot items to cards and fix up some colors

### PR DESCRIPTION
**Pull request by nickcharles**
@kenzshelley @matthewgrossman 

The coolest thing about this is that we get some tiny little shadows under the cards.

### CODE REVIEW

File: app/src/main/java/com/mmm/parq/fragments/HostHomeFragment.java [hunk 1]
```diff
@@ -37,7 +37,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
         final Toolbar toolbar = (Toolbar) v.findViewById(R.id.host_home_toolbar);
         toolbar.setTitle(R.string.host_home_titlebar);
-        toolbar.setTitleTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.white));
+        toolbar.setTitleTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), android.R.color.white));
```
Comment (nickcharles) [inline app/src/main/java/com/mmm/parq/fragments/HostHomeFragment.java:40]: Might not actually need this line anymore.

File: app/src/main/java/com/mmm/parq/fragments/HostNewSpotFragment.java [hunk 2]
```diff
@@ -48,7 +48,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
         final Toolbar toolbar = (Toolbar) v.findViewById(R.id.new_spot_toolbar);
         toolbar.setTitle(R.string.host_new_spot_titlebar);
-        toolbar.setTitleTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.white));
+        toolbar.setTitleTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), android.R.color.white));
```
Comment (nickcharles) [inline app/src/main/java/com/mmm/parq/fragments/HostNewSpotFragment.java:51]: Might not actually need this line anymore.

File: app/src/main/java/com/mmm/parq/fragments/HostSpotDetailsFragment.java [hunk 3]
```diff
@@ -57,7 +57,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
         final Toolbar toolbar = (Toolbar) v.findViewById(R.id.spot_details_toolbar);
         toolbar.setTitle(R.string.host_spot_details_titlebar);
-        toolbar.setTitleTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.white));
+        toolbar.setTitleTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), android.R.color.white));
```
Comment (nickcharles) [inline app/src/main/java/com/mmm/parq/fragments/HostSpotDetailsFragment.java:60]: Might not actually need this line anymore.

**Comment by kenzshelley** [comment]
lgtm

**Comment by nickcharles** [comment]
Don't merge this yet, concerned I may have messed up the colors.

**Comment by nickcharles** [comment]
False alarm, merging.

**Pull request merged**